src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: { books: booksReducer },
```

Comment:
- [ ] Kindly consider adding categories to your configure store as well because the purpose of the `configureStore` method is to create and configure the Redux store, which is responsible for managing the application's global state and that's what we need on both books and categories